README.md

```diff
@@ -0,0 +1,85 @@
+# MITOC Website
+
+This repository contains the source code of https://mitoc.mit.edu. 
+
+All members of the MITOC community are welcome to suggest edits by opening pull requests!
+
+If you'd like to change anything on the website, you can:
+- Read the rest of this short document to figure out how things work
+- Suggest a change on a file by clicking on the "Edit this file" button (see screenshot below)
+- Don't hesitate to contact the MITOC webasters if you need any help!
```

Comment:
May I suggest adding the email here too? `Don't hesitate to contact the MITOC webmasters (mitoc-webmasters@mit.edu) if you need any help`